README.md

```diff
@@ -1422,6 +1422,27 @@ features in the plugin configuration
 The list of privileges along with their description can be found in
 [security privileges](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-privileges.html).
 
+## Configuration - Elasticsearch Output Data Stream
+
+Since Elasticsearch 7.9, Data Streams was introduced.
+
+You can enable this feature by specifying `@type elasticsearch_data_stream`.
+
+```
+@type elasticsearch_data_stream
+data_stream_name test
+```
+
+When `@type elasticsearch_data_stream` is used, ILM default policy is set to specified data stream.
+Then, a matching index template is also created automatically.
```

Comment:
A matching index template is ambiguous for readers?
If not so, we write down with **the** not **a**:

Then, **the** matching index template is also created automatically.